Package.swift

```diff
@@ -53,6 +54,14 @@ let package = Package(
                 "XCBProtocol",
             ]
         ),
+        .target(
+            name: "XCBProtocol_12_5",
+            dependencies: [
+                "MessagePack",
+                "XCBProtocol",
+            ],
+            exclude: ["BUILD.bazel"]
```

Comment:
To match the previous versions, can this exclude be removed? It can get added back when reformatting/updating the package format in the subsequent PR.

Reply:
All set